lookout/style/format/visualizer/bblfsh/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM bblfsh/bblfshd:v2.9.0
+
+RUN bblfshd & sleep 1 \
+    && bblfshctl driver install javascript bblfsh/javascript-driver:v1.2.0
```

Comment:
I think it could be a good idea to store a variable/config with drivers requirements/versions somewhere - to avoid misbehavior after some modifications in Travis for example https://github.com/src-d/style-analyzer/blob/master/.travis.yml#L67 .

Reply:
alternative would be to use the image I made here (base bblfsh image + the drivers already installed) as the one used in style-analyzer in addition to the visualizer. WDYT @EgorBu @vmarkovtsev?

Reply:
For now, let's hack and slap shit together. I filed https://github.com/bblfsh/sdk/issues/317 for a proper solution.